packages/browser-wallet/src/popup/pages/Account/AccountSettings/ConnectedSites.tsx

```diff
@@ -1,46 +1,99 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useAtom, useAtomValue } from 'jotai';
 import { selectedAccountAtom, storedConnectedSitesAtom } from '@popup/store/account';
 import Button from '@popup/shared/Button';
 import { useTranslation } from 'react-i18next';
+import { popupMessageHandler } from '@popup/shared/message-handler';
+import { EventType } from '@concordium/browser-wallet-api-helpers';
 
 export default function ConnectedSites() {
     const { t } = useTranslation('account', { keyPrefix: 'settings.connectedSites' });
     const selectedAccount = useAtomValue(selectedAccountAtom);
     const [connectedSitesLoading, setConnectedSites] = useAtom(storedConnectedSitesAtom);
     const connectedSites = connectedSitesLoading.value;
+    const [openTabUrl, setOpenTabUrl] = useState<string>();
+
+    async function getCurrentOpenTabUrl() {
```

Comment:
Move this out of the component.

Reply:
I have done that in the following PR, so I will leave it be here to make merging easier: https://github.com/Concordium/concordium-browser-wallet/blob/3155f9ead7985a02db082ef5846caf44cdb1fc76/packages/browser-wallet/src/popup/shared/utils/tabs.ts.